CMakeLists.txt

```diff
@@ -235,3 +235,6 @@ if(NOT SKIP_INSTALL_FILES)
     install(FILES zlib.3 DESTINATION "${CMAKE_INSTALL_MANDIR}/man3")
     install(FILES ${ZLIB_PC} DESTINATION "${ZLIB_INSTALL_PKGCONFIG_DIR}")
 endif()
+
+set(CPACK_GENERATOR TGZ)
```

Comment:
The CPack generator shouldn't be set here. Let the consumer decide which method they want to used to package.

Reply:
That set only the default and may be changed with cpack -g ...

Reply:
ok